jp2_online/templates/administracion/side_nav.html

```diff
@@ -19,5 +19,19 @@
                 <li><a href="#">Signup</a></li>
             </ul>
         </li>
+        <li class="submenu">
+             <a href="#">
+                <i class="glyphicon glyphicon-list-alt"></i> Estudios Socioeconómicos
+                <span class="caret pull-right"></span>
+             </a>
+             <!-- Sub menu -->
+             <ul>
+                <li><a href="{% url 'administracion:main_estudios' 'aprobado' %}">Aprobados</a></li>
```

Comment:
Am not sure its a good idea to hardcode this into the view, use estudios_socieconomicos.models.Estudio.aprobado, maybe you need to get them from the view.